packages/web-components/src/components/structured-list/structured-list.scss

```diff
@@ -0,0 +1,9 @@
+//
+// Copyright IBM Corp. 2020, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//@import '@carbon/ibmdotcom-styles/scss/components/structured-list/structured-list';
+@import '../../../../styles/scss/components/structured-list/structured-list';
```

Comment:
Looks like the relative path is set in the import instead of the `@carbon/ibmdotcom-styles`, is there a reason for this?

`@import '@carbon/ibmdotcom-styles/scss/components/structured-liststructured-list;`

Reply:
Yeah, importing it this way during development allows live reloading. This is something we need to change before merging, but I want to leave this in place for now.